source/localizable/analytics/settings.html.md.erb

```diff
@@ -23,5 +23,29 @@ Name              | Type   | Required | Description
 ##### Example
 
 ~~~ javascript
-sa('session', 'connect', 'SA-0001-0001');
+<%= "#{flavor}_analytics" %>('session', 'connect', 'SA-0001-0001');
 ~~~
+
+## Renaming the `<%= "#{flavor}_analytics" %>` object
+
+It is possible that you'd like to add the Analytics tracking code to your page, but the `<%= "#{flavor}_analytics" %>`
```

Comment:
I think you should ✂️  `It is possible that you'd like to add the Analytics tracking code to your page,`, since the reader is already in that context.

Moreover, you are mentioning renaming twice in two consecutive sentences.

So, finally, you could simplify this to:
> In case the `skroutz_analytics` variable is already used in your page, `analytics.js` provides a mechanism to avoid the conflict with the Analytics tracking code.

> Specifically, you can rename the global `skroutz_analytics` object by changing
the final parameter passed to the minified function. You'll also need to update all invocations of the command
queue from `skroutz_analytics` to whatever name you choose.

Finally, you could revisit the title to of this section to reflect the problem (naming conflict), not the solution the problem.